recipe/meta.yaml

```diff
@@ -75,8 +77,9 @@ outputs:
         - glog
         - grpc-cpp {{ grpc_cpp }}
         - libprotobuf {{ protobuf }}
-        - llvmdev 10
-        - libutf8proc
+        - llvmdev 10  # [not s390x]
+        - libutf8proc # [not s390x]
+        - utf8proc # [s390x]
```

Comment:
As AnacondaRecipe of arrow-cpp and pyarrow has depdency on utf8proc instead of libutf8proc, I built that conda pkg/feedstock.